bin/oneoff/nces_data/export_schools

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../dashboard/config/environment'
+require 'optparse'
+
+CDO.log = Logger.new(STDOUT)
+
+options = {file: nil, test: false}
+OptionParser.new do |opts|
+  opts.banner = "Usage: #{File.basename(__FILE__)} [options]"
+  opts.on('-f', '--file FILE', 'The TSV filename.') do |file|
+    options[:file] = file
+  end
+  opts.on('-t', '--test', 'True for test/stub file.') do
+    options[:test] = true
+  end
+  opts.on('-h', '--help', 'Prints this help') do
+    puts opts
+    exit
+  end
+end.parse!
+CDO.log.info "Called with options: #{options}"
+
+if options[:test]
+  School.write_to_csv(options[:file]) do
+    ids = %w(
+      10000200277
+      10000201402
+      10000500870
+      10000500871
+      20000100206
+      20000100207
+      20000300216
+      20000300217
+      60000113121
+      60000113717
+      60001411118
+      60001411746
+      480000407173
+      480000409116
+      480001407174
+      480001410415
+    )
```

Comment:
Should we parse the ids from the existing [schools.tsv](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/fixtures/schools.tsv) file instead of hard-coding them here? Or alternatively, just take the top n? I don't think we care about which specific ids, just that we have a small (quick to import) subset.

Reply:
Ya, I was thinking about that as well. There are some tests that rely on specific entries which is why I wanted to hard code the IDs. Loading from the original source file sounds better.